src/frontend/Sider/Behandling/Brev/lagHtmlFelt.ts

```diff
@@ -0,0 +1,49 @@
+import { BeregningsresultatTilsynBarn } from '../../../typer/vedtak';
+import { formaterÅrMåned } from '../../../utils/dato';
+import { formaterTallMedTusenSkille } from '../../../utils/fomatering';
+
+const borderStylingCompact = 'border: 1px solid black; padding: 3px 2px 3px 5px;';
+const borderStyling = 'border: 1px solid black; padding: 3px 10px 3px 5px;';
+
+export const lagHtmlFelt = (beregningsresultat: BeregningsresultatTilsynBarn | undefined) => {
+    const variabelBeregningstabellId = '02408de1-5ad8-44e9-a004-51f677d6ebab';
+
+    return {
+        [variabelBeregningstabellId]: lagBeregningstabell(beregningsresultat),
+    };
+};
+const lagBeregningstabell = (beregningsresultat?: BeregningsresultatTilsynBarn): string => {
+    return `<table style="margin-left: 2px; margin-right: 2px; border-collapse: collapse; ${borderStylingCompact}">
```

Comment:
Hvis `beregningsresultat` er undefined så burde man kanskje ikke svare med en tabell? 
Vi har alltid et beregningsresultat fra et innvilget vedtak (men typen har ikke det fordi vi bruker den samme for request og response)